docs/api.adoc

```diff
@@ -56,6 +56,7 @@ Return the list of all not hidden (by default) accounts from the keystore.
     * `accounts` (Array)
     ** `account` (Object) - an account
     *** `address` (String) - hex-encoded 20 bytes public address
+    *** `is_hd` (Boolean) - flag to distinguish normal accounts from HD wallet accounts
```

Comment:
A little bit cryptic, not good for API, probably better to use `hardware_wallet`?